tools/bin/lib/.gitignore

```diff
@@ -2,7 +2,3 @@ netperf
 netserver
 stream
 Crypto
```

Comment:
+1. Good catch for the whole issue.

By the way, tools/Makefile seems to be buggy. See the following patch.
This misleads me also. @ edespino and @radarwave ,
how about the patch below? If yes, Ed you probably could
add this change in you patch?

diff --git a/tools/Makefile b/tools/Makefile
index 172f6f6..3eb49f6 100644
--- a/tools/Makefile
+++ b/tools/Makefile
@@ -27,6 +27,7 @@ include $(top_builddir)/src/Makefile.global

 all:
    ${MAKE} -C gpnetbench $@
+   ${MAKE} -C bin $@

 install :
    ${MAKE} -C bin $@